q2_quality_filter/plugin_setup.py

```diff
@@ -26,13 +39,28 @@
     artifact_format=QualityFilterStatsDirFmt)
 
 plugin.methods.register_function(
-    function=q2_quality_filter.basic,
+    function=q2_quality_filter.q_score,
     inputs={'demux': SampleData[SequencesWithQuality]},
+    input_descriptions={
+        'demux': 'The per-sample sequence data to quality filter'
+    },
     parameters={
-        'minimum_quality': qiime2.plugin.Int,
+        'min_quality': qiime2.plugin.Int,
         'quality_window': qiime2.plugin.Int,
         'min_length_fraction': qiime2.plugin.Float,
-        'maximum_ambiguous': qiime2.plugin.Int
+        'max_ambiguous': qiime2.plugin.Int
+    },
+
+    # descriptions adapted from QIIME 1.9.1's split_libraries_fastq.py
+    parameter_descriptions={
+        'min_quality': 'The maximum unacceptable PHRED score',
+        'quality_window': ('The maximum number of low quality base calls to '
+                           'observe before truncating'),
+        'min_length_fraction': ('The minimum number of consecutive high '
```

Comment:
That is more accurate, will change. The text used here is derived from:

https://github.com/biocore/qiime/blob/master/scripts/split_libraries_fastq.py#L102-L105